eng/pipelines/templates/variables/rust.yml

```diff
@@ -0,0 +1,3 @@
+variables:
```

Comment:
Why not just add these to globals.yml?

Reply:
I thought about it, but wasn't entirely sure it was imported everywhere and didn't want to worry about the ramifications of doing so. If you think it's safer, I'm fine doing that. Note for example, though, it's not in https://github.com/Azure/azure-sdk-for-rust/pull/2260/files#diff-8846d7741afb575539571d7b46758247229daaa4336751c6b3e743fd284426fa. I imagine that's probably why @analogrelay did it that way as well. `global.yml` didin't seem truly global.